src/huggingface_hub/file_download.py

```diff
@@ -100,15 +101,15 @@ def are_symlinks_supported(cache_dir: Union[str, Path, None] = None) -> bool:
                 # Likely running on Windows
                 _are_symlinks_supported_in_dir[cache_dir] = False
 
-                if not os.environ.get("DISABLE_SYMLINKS_WARNING"):
+                if not HF_HUB_DISABLE_SYMLINKS_WARNING:
```

Comment:
small advise (not needed for this PR and not urgent at all)

To make testing easier, it's nice to be able to modify the constants at runtime (e.g. in the `datasets` CI)

For this to work as expected, you can just point to the variables names using `constants.<name>` instead of importing `<name>` when the file is imported.

Reply:
Hmm, true yes. For now what I do is to patch the constant individually in the test e.g.

```py
@patch("huggingface_hub.utils._headers.HF_HUB_DISABLE_IMPLICIT_TOKEN", True):
```